src/main/java/lotto/constant/OutputMessage.java

```diff
@@ -0,0 +1,19 @@
+package lotto.constant;
+
+public class OutputMessage {
+    public static final String PURCHASE_AMOUNT_REQUEST = "구입금액을 입력해 주세요.";
+    public static final String LOTTO_PURCHASE_COUNT_MESSAGE = "개를 구매했습니다.";
+    public static final String WINNING_NUMBER_REQUEST = "당첨 번호를 입력해 주세요.";
+    public static final String BONUS_NUMBER_REQUEST = "보너스 번호를 입력해 주세요.";
+    public static final String MESSAGE_WINNING_STATISTICS = "당첨 통계";
+    public static final String LINE = "---";
+
+    public static final String LOTTO_RESULT_FIFTH = "3개 일치 (5,000원)";
+    public static final String LOTTO_RESULT_FOURTH = "4개 일치 (50,000원)";
```

Comment:
String.format을 이용하면 선언한 메시지의 개수를 줄일 수 있을 것 같습니다!
```suggestion
    public static final String LOTTO_RESULT_FOURTH = "%s개 일치 (%d원)";
```

Reply:
서식 지정자를 이용해서 하니, 재사용할 수 있겠네요!
짚어주셔서 감사합니다 :)